src/sidebar/services/groups.js

```diff
@@ -4,15 +4,19 @@ import { combineGroups } from '../util/groups';
 import { awaitStateChange } from '../util/state';
 import { watch } from '../util/watch';
 
+/** @typedef {import('../../types/api').Group} Group */
+
 const DEFAULT_ORG_ID = '__default__';
 
 /**
  * FIXME: There is almost assuredly a better way to handle a fallback, default logo
  */
 const DEFAULT_ORGANIZATION = {
   id: DEFAULT_ORG_ID,
+  name: '__DEFAULT__',
```

Comment:
`name` is a mandatory parameter of organizations returned from the server, so I added an unused value here.